Add payment request fields for partial payment (#699)
centrapay/centrapay-docs: legacy/src/api/payment-requests/payment-requests.md

| cancellationReason   | String             | The reason that the payment request was cancelled. See [Cancellation Reasons](#cancellation-reasons) for possible values.                               |
| preAuth              | Boolean            | Flag to indicate the if the request is a Pre Auth for supported [Asset Types][].                                                                        |
| preAuthExpiresAt     | {% dt Timestamp %} | Pre Auth completions and releases will be accepted until this time.                                                                                     |
| preAuthStatus        | String             | Describes which state a Pre Auth Payment Request is in. Valid values are "authorized", or "released".                                                   |
| taxNumber            | [Tax Number][]     | The value-added tax configuration for the [Business][] that the [Merchant][] belongs to.                                                                |
| remainingAmount      | {% dt BigNumber %} | The amount of the payment request which has not been paid for.                                                                                          |

> | basketAmount         | {% dt BigNumber %} | The total amount of the payment including payments made outside of Centrapay that cannot be paid for using Centrapay                                    |

Reviewer: Maybe something like this
```suggestion
The total amount of the transaction including non Centrapay payment methods
```

Reviewer 2: sounds good
